### PR TITLE
[Merged by Bors] - api: v2alpha1 account service

### PR DESCRIPTION
**Pull request by kacpersaw**
```
service AccountService {
  rpc List(AccountRequest) returns (AccountList);
}
```

https://github.com/spacemeshos/api/pull/337

### CODE REVIEW

File: sql/accounts/accounts.go [hunk 1]
```diff
@@ -169,3 +170,35 @@ func Revert(db sql.Executor, after types.LayerID) error {
...
+			account.Layer = types.LayerID(uint32(stmt.ColumnInt64(3)))
+			if stmt.ColumnLen(4) > 0 {
+				var template types.Address
+				stmt.ColumnBytes(4, template[:])
+				account.TemplateAddress = &template
+				account.State = make([]byte, stmt.ColumnLen(5))
+				stmt.ColumnBytes(5, account.State)
+			}
+			fn(&account)
+			return true
```
Comment (poszu) [inline sql/accounts/accounts.go]:
```suggestion
			return fn(&account)
```

How about adding a UT to see if iteration is stopped when `fn` returns false?

**Review by poszu — CHANGES_REQUESTED** [review]

**Review by poszu — APPROVED** [review]

**Comment by kacpersaw** [comment]
bors merge

**Comment by kacpersaw** [comment]
bors merge

**Comment by kacpersaw** [comment]
bors merge

**Pull request closed without merging**